packages/terra-table/src/subcomponents/Row.jsx

```diff
@@ -176,6 +190,9 @@ function Row(props) {
             onCellSelect={onCellSelect}
             height={height}
             rowMinimumHeight={rowMinimumHeight}
+            firstRowId={firstRowId}
+            lastRowId={lastRowId}
```

Comment:
The row already knows the row id.  It should not have to pass this information to the cell, correct?

Reply:
https://github.com/cerner/terra-framework/pull/1985#discussion_r1483944966